tasks/seal/poller_commit_msg.go

```diff
@@ -154,29 +153,13 @@ func (s *SealPoller) pollStartBatchCommitMsg(ctx context.Context, tasks []pollTa
 }
 
 func (s *SealPoller) sendCommitBatch(ctx context.Context, spid int64, sectors []int64) {
-	if len(sectors) < miner.MinAggregatedSectors {
-		for i := range sectors {
-			s.pollers[pollerCommitMsg].Val(ctx)(func(id harmonytask.TaskID, tx *harmonydb.Tx) (shouldCommit bool, seriousError error) {
-				n, err := tx.Exec(`UPDATE sectors_sdr_pipeline SET task_id_commit_msg = $1 WHERE sp_id = $2 AND sector_number = $3 AND task_id_commit_msg IS NULL`, id, spid, sectors[i])
-				if err != nil {
-					return false, xerrors.Errorf("update sectors_sdr_pipeline: %w", err)
-				}
-				if n != 1 {
-					return false, xerrors.Errorf("expected to update 1 row, updated %d", n)
-				}
-
-				return true, nil
-			})
-		}
-	}
```

Comment:
@LexLuthr I really don't get why this is here, it's not needed, right?

Also at minimum it should return early

Reply:
That was for aggregation and non aggregate approach I was doing earlier before we moved to always batch but not always aggregate strategy. This can be removed.